api2/pkg/apps/iam/static.go

```diff
@@ -0,0 +1,230 @@
+			Locale:           "en",
+			LongFormulation:  "Email",
+			ShortFormulation: "Email",
+		},
+	},
+	IsPersonallyIdentifiableInfo: true,
+	PartyTypeIDs: []string{
+		IndividualPartyType.ID,
+	},
+}
+
+// StaffRelationshipType represents the built-in Staff relationship type
+var StaffRelationshipType = RelationshipType{
+	ID:              "53478121-23af-4ed8-a367-2e0de6d60271",
+	Name:            "staff",
+	FirstPartyRole:  "Is working for",
+	SecondPartyRole: "Has staff",
+	Rules: []RelationshipTypeRule{
+		{
+			PartyTypeRule: &PartyTypeRule{
+				FirstPartyTypeID:  IndividualPartyType.ID,
+				SecondPartyTypeID: OrganizationPartyType.ID,
+			},
+		},
+	},
+}
+
+var IndividualPartyType = PartyType{
+	ID:        "a842e7cb-3777-423a-9478-f1348be3b4a5",
+	Name:      "Individual",
+	IsBuiltIn: true,
+}
+
+var HouseholdPartyType = PartyType{
+	ID:        "d38a7085-7dff-4730-8be1-7c9d92a20cc3",
+	Name:      "Household",
+	IsBuiltIn: true,
+}
+
+var OrganizationPartyType = PartyType{
+	ID:        "09a7eef9-3f23-4c40-86f4-9b9440c56c6f",
+	Name:      "Organization",
+	IsBuiltIn: true,
+}
+
+var TeamPartyType = PartyType{
+	ID:        "dacd6e08-3e3d-495b-8655-ea1d8e822cf3",
+	Name:      "Team",
+	IsBuiltIn: true,
+}
+
+var TeamNameAttribute = Attribute{
+	ID:                           "18f410a3-6fde-45ce-80c7-fc5d92b85870",
+	Name:                         "teamName",
+	PartyTypeIDs:                 []string{TeamPartyType.ID},
+	IsPersonallyIdentifiableInfo: false,
+	Translations: []AttributeTranslation{
+		{
+			Locale:           "en",
+			ShortFormulation: "Team name",
+			LongFormulation:  "Team name",
+		},
+	},
+}
+
+var MembershipRelationshipType = RelationshipType{
+	ID:              "69fef57b-b37f-4803-a5fb-47e05282ac84",
+	IsDirectional:   true,
+	Name:            "teamMembership",
+	FirstPartyRole:  "Is member of team",
+	SecondPartyRole: "Has team member",
+	Rules: []RelationshipTypeRule{
+		{
+			&PartyTypeRule{
+				FirstPartyTypeID:  IndividualPartyType.ID,
+				SecondPartyTypeID: TeamPartyType.ID,
+			},
+		},
+	},
+}
+
+var HeadOfHouseholdRelationshipType = RelationshipType{
+	ID:              "de887604-9ce9-4fdc-af6b-602091a17913",
+	IsDirectional:   true,
+	Name:            "headOfHousehold",
+	FirstPartyRole:  "Is head of household of",
+	SecondPartyRole: "Has for head of household",
+	Rules: []RelationshipTypeRule{
+		{
+			PartyTypeRule: &PartyTypeRule{
+				FirstPartyTypeID:  IndividualPartyType.ID,
+				SecondPartyTypeID: HouseholdPartyType.ID,
+			},
+		},
+	},
+}
+
+var SpousalRelationshipType = RelationshipType{
+	ID:              "76376c69-ce06-4e06-b603-44c145ddf399",
+	IsDirectional:   false,
+	Name:            "spousal",
+	FirstPartyRole:  "Is spouse of",
+	SecondPartyRole: "Is spouse of",
+	Rules: []RelationshipTypeRule{
+		{
+			PartyTypeRule: &PartyTypeRule{
+				FirstPartyTypeID:  IndividualPartyType.ID,
+				SecondPartyTypeID: IndividualPartyType.ID,
+			},
+		},
+	},
+}
+
+var FilialRelationshipType = RelationshipType{
```

Comment:
```suggestion
var SiblingRelationshipType = RelationshipType{
```
'Filial' suggest a "son of" or "daughter of" relationship.